queue/queue_header.php

```diff
@@ -1,9 +1,14 @@
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script src="/queue/queue_header.js"></script>
 <div class="container-fluid dashboard bg-light py-3" ng-cloak>
 	<div class="d-flex flex-row justify-content-between align-items-center">
```

Comment:
I'm wondering if this should have ```d-flex flex-column flex-sm-row``` rather than ```d-flex flex-row``` so that the header elements stack on top of each other on small screens.